CMakeLists.txt

```diff
@@ -62,7 +62,7 @@ if (${BUILD_PYTHON_BINDINGS})
     FILE(COPY carma_clock_py/python_wrapper_test.py DESTINATION .)
     add_test(NAME test_carma_clock_python_module_binding COMMAND ${PYTHON_EXECUTABLE} python_wrapper_test.py )
     # set cpack depedencies
-    set(CPACK_DEBIAN_PACKAGE_DEPENDS carma-clock-1 python3-dev)
+    set(CPACK_DEBIAN_PACKAGE_DEPENDS python3-dev)
```

Comment:
Something I noticed. This is the carma-clock-1 package. It can't depend on itself